### PR TITLE
Add documentation for json serialization

### PR DESCRIPTION
**Pull request by HornMichaelS**
I recently found the .jsonObject property very useful for serializing some Apollo iOS data in a format that was compatible with RealmSwift. However, I was only able to find out about the property by digging through some issues on GitHub. So I thought it might be helpful to have it officially documented somewhere. 

If there's a better place for this information, or if it should remain undocumented, please let me know. Thanks for all your work on this!

### CODE REVIEW

**Comment by designatednerd** [comment]
@HornMichaelS Would you like to fix the conflicts or would you prefer I do the honors?

**Comment by HornMichaelS** [comment]
Hi @designatednerd, thanks for taking a look at this! I'll go ahead and fix those conflicts

**Comment by HornMichaelS** [comment]
Hi @designatednerd, thanks for taking a look at this! I'll go ahead and fix those conflicts

**Comment by HornMichaelS** [comment]
Hi @designatednerd, thanks for taking a look at this! I'll go ahead and fix those conflicts

**Comment by HornMichaelS** [comment]
Hi @designatednerd, thanks for taking a look at this! I'll go ahead and fix those conflicts

**Comment by HornMichaelS** [comment]
Alright, should be good to go now! I made a couple further changes, but they're fairly trivial. Changed the syntax of my subheading to markdown, and removed a sentence in the section above mine that was duplicated for some reason

File: docs/source/fetching-queries.md [hunk 1]
```diff
@@ -118,4 +118,24 @@ By default, Apollo constructs queries and sends them to your graphql endpoint us
...
+  // Serialize the response as JSON
+  let json = data.jsonObject
+  let serialized = try! JSONSerialization.data(withJSONObject: json, options: [])
+  
+  // Deserialize the response
+  let deserialized = try! JSONSerialization.jsonObject(with: serialized, options: []) as! JSONObject
+  let heroAndFriendsNames = try! HeroAndFriendsNamesQuery.Data(jsonObject: deserialized)
+}
+```
+```
```
Comment (designatednerd) [inline docs/source/fetching-queries.md]: looks like an extra set of backticks snuck in here

Comment (HornMichaelS) [reply docs/source/fetching-queries.md]: Thanks for catching that! It's fixed now

**Pull request merged**